src/editor.js

```diff
@@ -490,13 +597,19 @@ class Editor {
     this.updateFileHints_();
   }
 
+  showTemplates_() {
+    this.state_.isTemplateLightboxDisplayed = !this.state_
+      .isTemplateLightboxDisplayed;
+  }
+
   selectViewport_({target}) {
     const {value} = target;
 
     // If viewport is changed to 'full', the view will display an error message
     // instead of the preview.
     // TODO: Make 'full' special and add custom sizing
     this.state_.viewportId = validViewportId(value);
+    this.updatePreview_();
```

Comment:
Why `updatePreview_` here?

Reply:
Otherwise text/other elements don't resize properly and end up getting cut off in smaller viewports.

Reply:
Ah, that might be an `amp-story` bug. Let's remove this `updatePreview_` and add a `TODO` in its place explaining the issue and to investigate later.